### PR TITLE
manage cannon registry permissions through gitops

### PR DESCRIPTION
**Pull request by dbeal-eth**

### CODE REVIEW

File: tomls/pools/spartan-council.toml [hunk 1]
```diff
@@ -6,7 +6,6 @@
 
 [invoke.createScPool]
 target = ["system.CoreProxy"]
-from = "<%= settings.deployer %>"
```
Comment (noisekit) [inline tomls/pools/spartan-council.toml:9]: won't this redeploy pool?
also do we have permissions added to deployer in some places (permissions.toml)?

File: omnibus-mainnet.toml [hunk 2]
```diff
@@ -86,7 +89,7 @@ defaultValue = "1500000000000000000"
 defaultValue = "86400"
 
 [setting.deployer]
-defaultValue = "0xEde8a407913A874Dd7e3d5B731AFcA135D30375E"
+defaultValue = "0x1C8236B406911A376369e33D39189F1b4B39F27D"
```
Comment (noisekit) [inline omnibus-mainnet.toml:92]: will this change lead to some new invocations?

File: tomls/omnibus-mainnet/cannon-permissions.toml [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+        "0x6cd3f878852769e04A723A5f66CA7DD4d9E38A6C",
+        "<%= settings.deployer %>",
+    ],
+]
+
+[invoke.authorizedPublishers_chainlink]
+target = ["cannonRegistry.Proxy"]
+fromCall.func = "getPackageOwner"
+fromCall.args = ["<%= formatBytes32String('chainlink') %>"]
+func = "setAdditionalPublishers"
```
Comment (noisekit) [inline tomls/omnibus-mainnet/cannon-permissions.toml:518]: how about removing publishers?

File: tomls/pools/spartan-council.toml (hunk 1 above)
Comment (dbeal-eth) [reply tomls/pools/spartan-council.toml:9]: does not

File: tomls/permit-deniers.toml [hunk 4]
```diff
@@ -13,7 +13,7 @@ args = [
         "0x599e835cbFC903eF09f3Dd5E08D1cF63c32AF8d8",
         "0x347c3190bD015FBD0e47fb90AA4917138A8A32FE",
         "0x562948111d50BF039A39Eea48D127f2Ae51ddF02",
-        "0x1C8236B406911A376369e33D39189F1b4B39F27D",
+        "<%= settings.deployer %>",
```
Comment (noisekit) [inline tomls/permit-deniers.toml:16]: should we consider not having a depolyer as a denier at all maybe?

File: tomls/omnibus-mainnet/cannon-permissions.toml (hunk 3 above)
Comment (dbeal-eth) [reply tomls/omnibus-mainnet/cannon-permissions.toml:518]: its a "setter" so this function replaces any old publishers that may have existed.

File: tomls/permit-deniers.toml (hunk 4 above)
Comment (dbeal-eth) [reply tomls/permit-deniers.toml:16]: its been a discussion before. not sure if now is a good time to be rethinking this.

File: tomls/omnibus-mainnet/cannon-permissions.toml (hunk 3 above)
Comment (noisekit) [reply tomls/omnibus-mainnet/cannon-permissions.toml:518]: aaah ok. `additional` word confused me.

File: tomls/permit-deniers.toml (hunk 4 above)
Comment (dbeal-eth) [reply tomls/permit-deniers.toml:16]: considering how few pdao signers we have and the minimal concern if someone were to deny the system, doesn't seem too important to remove at this time.

**Pull request merged**